src/uct/api/uct.h

```diff
@@ -927,41 +917,52 @@ ucs_status_t uct_iface_get_address(uct_iface_h iface, uct_iface_addr_t *addr);
 int uct_iface_is_reachable(const uct_iface_h iface, const uct_device_addr_t *dev_addr,
                            const uct_iface_addr_t *iface_addr);
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief  Wakeup event types.
+ */
+enum uct_wakeup_event_types {
+    UCT_WAKEUP_TX_COMPLETION   = UCS_BIT(0),
```

Comment:
pls set following events. we need also signaled tx completion to allow specific signaling for protocols such as rendezvous.
```
UCT_WAKEUP_TX_COMP,
UCT_WAKEUP_TX_SIGNALED_COMP,
UCT_WAKEUP_RX_AM,
UCT_WAKEUP_RX_SIGNALED_AM,
UCT_WAKEUP_TX
```